src/fides/api/ops/models/application_config.py

```diff
@@ -82,3 +112,86 @@ def get_api_set_config(cls, db: Session) -> Dict[str, Any]:
         if config_record:
             return config_record.api_set
         return {}
+
+    @classmethod
+    def get_config_set_config(cls, db: Session) -> Dict[str, Any]:
+        """
+        Utility method to get the config_set config dict
+
+        An empty `dict` will be returned if no config settings have been set through config
+        """
+        config_record = db.query(cls).first()
+        if config_record:
+            return config_record.config_set
+        return {}
+
+    @classmethod
+    def set_api_set_config(
+        cls, db: Session, api_set_dict: Dict[str, Any]
+    ) -> ApplicationConfig:
+        """
+        Utility method to set the `api_set` column on the `applicationconfig`
+        db record with the provided dictionary of data.
+
+        Updates are *merged* with any existing `api_set` data in the db.
+        """
+        return cls.create_or_update(db, data={"api_set": api_set_dict})
+
+    @classmethod
+    def set_config_set_config(
```

Comment:
😆 it is, though i definitely paused while writing it. i call the column `config_set` because it contains the config properties set by traditional "config" mechanisms. but i really hate the name. any suggestions?

even if i don't rename the column, i could rename this method. let me think about something a bit less of a mouthful.

Reply:
Now I see your struggle, the other name that comes to mind is `set_set_config` which is no better :smile: